packages/storefront/tests/e2e/specs/navigation.e2e.ts

```diff
@@ -69,14 +69,15 @@ it.each(cases.map((segments) => [segments.filter(Boolean).join(' > '), ...segmen
 for (const [category, pages] of Object.entries(STOREFRONT_CONFIG)) {
   for (const [page, tabs] of Object.entries(pages)) {
     ((category: string, page: string) => {
-      it(`should navigate to "${category} > ${page}"`, async () => {
-        console.log(`${category} > ${page}`);
+      const [tab] = Object.keys(tabs);
```

Comment:
Isn't `tabs` an array? And `Object.keys()` returns the indexes, e.g. `0`, `1`, etc.?
If yes, then calling it `tab` is a bit confusing and will always be `0` since the first index is destructured.